saleor/graphql/account/mutations/permission_group.py

```diff
@@ -107,118 +155,106 @@ class Meta:
         description = "Update permission group."
         model = auth_models.Group
         permissions = (AccountPermissions.MANAGE_STAFF,)
-        error_type_class = BulkAccountError
-        error_type_field = "bulk_account_errors"
+        error_type_class = PermissionGroupError
+        error_type_field = "permission_group_errors"
 
+    @classmethod
+    @transaction.atomic
+    def _save_m2m(cls, info, instance, cleaned_data):
+        cls.update_group_permissions_and_users(instance, cleaned_data)
 
-class PermissionGroupDelete(ModelDeleteMutation):
-    class Arguments:
-        id = graphene.ID(description="ID of the group to delete.", required=True)
-
-    class Meta:
-        description = "Delete permission group."
-        model = auth_models.Group
-        permissions = (AccountPermissions.MANAGE_STAFF,)
-        error_type_class = AccountError
-        error_type_field = "account_errors"
+    @classmethod
+    def update_group_permissions_and_users(
+        cls, group: auth_models.Group, cleaned_input: dict
+    ):
+        if "add_users" in cleaned_input:
+            group.user_set.add(*cleaned_input["add_users"])
+        if "remove_users" in cleaned_input:
+            group.user_set.remove(*cleaned_input["remove_users"])
+
+        if "add_permissions" in cleaned_input:
+            group.permissions.add(*cleaned_input["add_permissions"])
+        if "remove_perissions" in cleaned_input:
+            remove_permissions = get_permissions(cleaned_input["remove_perissions"])
+            group.permissions.remove(*remove_permissions)
 
+    @classmethod
+    def clean_input(
+        cls, info, instance, data,
+    ):
+        if not can_user_manage_group(info.context.user, instance):
+            error_msg = "You can't manage group with permissions out of your scope."
+            code = PermissionGroupErrorCode.OUT_OF_SCOPE_PERMISSION.value
+            raise ValidationError({None: ValidationError(error_msg, code)})
```

Comment:
Since the error is not assigned to a field, couldn't we just do this?
```suggestion
            raise ValidationError(error_msg, code)
```